src/main/resources/schema/ans/0.10.10/traits/trait_variants.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "additionalProperties": false,
+  "description": "Holds variant content metadata, including placeholder IDs for use within 'content_elements' and mapping from website IDs to variant IDs",
+  "id": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.10/traits/trait_variants.json",
+  "properties": {
+    "additional_properties": {
+      "$ref": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.10/traits/trait_additional_properties.json"
+    },
+    "placeholder_ids": {
```

Comment:
What's the value added here?  It's helpful, but also redundant since you have them in the "content"elements" of the hub story.

Reply:
Those won't be present until the variant content exists.  This defines them first so they can be used when the content is created.  Additionally, the variant content isn't guaranteed to override all of the zones, so this just defined the zones that are possible.

Reply:
makes sense.